proposals/lip-0048.md

```diff
@@ -62,8 +62,6 @@ We use the definition of the following types:
 
 | Name          | Type   | Validation                                                                                                                                                                                                                                                                                    | Description                                          |
 | ------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
-| `Transaction` | object | Must follow the `transactionSchema` schema defined in [LIP 0068](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0068.md) with the only difference that `params` property is not serialized and contains the values of parameters of `paramsSchema` for the corresponding transaction. | An object representing a non-serialized transaction. |
-| `CCM`         | object | Must follow the `crossChainMessageSchema` schema defined in [LIP 0049][lip-0049#ccmschema].                                                                                                                                                                                                   | The type of cross-chain messages.                    |
```

Comment:
As we just have one type `CCM` that is the same everywhere and I don't see any risk of confusing about it, I would prefer to avoid this redundancy in LIPs.

Reply:
ok